tests/runner/src/main.rs

```diff
@@ -27,13 +27,18 @@ async fn main() {
         "".to_string()
     };
 
+    #[cfg(windows)]
+    let data_home = std::env::temp_dir();
+    #[cfg(not(windows))]
+    let data_home = std::path::PathBuf::from("/tmp");
```

Comment:
question: Could we also use `std::env::temp_dir()` on UNIX?

Reply:
On macos, the `temp_dir()` is not always "/tmp".